tests/test_input.py

```diff
@@ -0,0 +1,291 @@
+#
+# Copyright 2019 the original author or authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""
+Test input
+"""
+
+import unittest
+
+import numpy as np
+import pygame
+
+from qpong.utils.level import Level
+from qpong.utils.scene import Scene
+from qpong.utils.ball import Ball
+from qpong.utils.input import Input
+
+from qpong.model import circuit_node_types as node_types
+
+from qpong.utils.parameters import WINDOW_SIZE
+
+
+class TestLevel(unittest.TestCase):
```

Comment:
Add tests for joysticks for the future. Can you please open an issue to track that?